### ISSUE TITLE
About chest dataset

### ISSUE THREAD

**Issue by Jiawei0o0** [ConstantinSeibold/SGL#2]
Thanks for sharing your work and code. Will you update your code about the chest X-ray dataset?  By the way, can you share your train list, Val list, and test list about the chest dataset? 
Looking forward to your response! Thanks again!

**Comment by ConstantinSeibold** [comment]
Hi VoooWei,

@Question1 "Will you update your code about the chest X-ray dataset?"

Due to the experiments on ChestXRay14 being encorporated into a larger framework for various tasks, I have seen it as a large effort to extract the minimal working experiments on the dataset due to different dependecies within. Thus, we made the MNIST experiments available, however, this contains the same utilized loss function which can be integrated into a common ResNet architecture.

As of now, it is not planned to expand this repository with the XRay experiments in near future due to time constraints on my part.

@Question2 " By the way, can you share your train list, Val list, and test list about the chest dataset?"

To load the ChestXRay14 data, I downloaded the official lists from https://nihcc.app.box.com/v/ChestXray-NIHCC. I split the train_val_list patientwise. The test set is the one provided in test_list. Then, I merged all the info into the DataEntry_2017 'csv'-file. 

[Data_Entry_2017_updated_compact.zip](https://github.com/ConstantinSeibold/SGL/files/5615118/Data_Entry_2017_updated_compact.zip)

If you notice any irregularities or  have any other questions, please let me know.
